kronos-java/src/main/java/com/lyft/kronos/internal/ntp/SntpClient.java

```diff
@@ -69,6 +75,10 @@ public SntpClient(Clock deviceClock, DnsResolver dnsResolver, DatagramFactory da
      * @throws IOException network error
      */
     public Response requestTime(String host, Long timeout) throws IOException {
+        return requestTime(host, timeout, true);
```

Comment:
Looks like we only pass `true` here, but only test with the parameter `false`. Maybe we should spend some time updating this class more comprehensively to match the new Google code or parsing out which pieces we need.

Reply:
The reason I did this is so I can bypass the new validation code in unit test.

Reply:
okay I took out this extra argument and fixed the unit test.